src/server/validations/festivals.js

```diff
@@ -27,6 +27,7 @@ export default {
     [Segments.PARAMS]: {
       ...paginationValidation,
       orderKey: Joi.string().valid('id', 'createdAt', 'updatedAt', 'title'),
+      query: Joi.string(),
```

Comment:
Can this be an actual JSON object? This will be easier to verify and does not need to be parsed then.